manpages/fsck.fat.8.in

```diff
@@ -103,125 +103,129 @@ The other is the legacy Atari variant used on Atari ST.
 .SH OPTIONS
-.IP "\fB\-a\fR" 4
+.IP "\fB\-a\fP" 4
 Automatically repair the filesystem.
 No user intervention is necessary.
 Whenever there is more than one method to solve a problem, the least
 destructive approach is used.
-.IP "\fB\-A\fR" 4
+.IP "\fB\-A\fP" 4
 Select using the Atari variation of the FAT filesystem if that isn't active
 already, otherwise select standard FAT filesystem.
-This is selected by default if \fBmkfs.fat\fR is run on 68k Atari Linux.
-.IP "\fB-b\fR" 4
+This is selected by default if \fBmkfs.fat\fP is run on 68k Atari Linux.
+.IP "\fB-b\fP" 4
 Make read-only boot sector check.
-.IP "\fB-c\fR \fIPAGE\fR" 4
-Use DOS codepage \fIPAGE\fR to decode short file names.
+.IP "\fB-c\fP \fIPAGE\fP" 4
+Use DOS codepage \fIPAGE\fP to decode short file names.
 By default codepage 850 is used.
-.IP "\fB\-d\fR \fIPATH\fR" 4
+.IP "\fB\-d\fP \fIPATH\fP" 4
 Delete the specified file.
 If more than one file with that name exist, the first one is deleted.
 This option can be given more than once.
-.IP "\fB\-f\fR" 4
+.IP "\fB\-f\fP" 4
 Salvage unused cluster chains to files.
 By default, unused clusters are added to the free disk space except in auto mode
-(\fB\-a\fR).
-.IP "\fB\-l\fR" 4
+(\fB\-a\fP).
+.IP "\fB\-l\fP" 4
 List path names of files being processed.
-.IP "\fB\-n\fR" 4
-No\-operation mode: non\-interactively check for errors, but don't write
+.IP "\fB\-n\fP" 4
+No-operation mode: non-interactively check for errors, but don't write
 anything to the filesystem.
-.IP "\fB\-p\fR" 4
-Same as \fB\-a\fR, for compatibility with other *fsck.
-.IP "\fB\-r\fR" 4
+.IP "\fB\-p\fP" 4
+Same as \fB\-a\fP, for compatibility with other *fsck.
+.IP "\fB\-r\fP" 4
 Interactively repair the filesystem.
 The user is asked for advice whenever there is more than one approach to fix an
 inconsistency.
 This is the default mode and the option is only retained for backwards
 compatibility.
-.IP "\fB\-S\fR" 4
+.IP "\fB\-S\fP" 4
 Consider short (8.3) file names with spaces in the middle to be invalid, like
 previous versions of this program did.
 While such file names are not forbidden by the FAT specification, and were never
 treated as errors by Microsoft file system checking tools, many DOS programs are
 unable to handle files with such names. Using this option can make them
 accessible to these programs.
 .IP "" 4
-Short file names which \fIstart\fR with a space are considered invalid
+Short file names which \fIstart\fP with a space are considered invalid
 regardless of this option's setting.
 .IP "" 4
-Previous versions of this program exceptionally treated "EA DATA . SF" and
-"WP ROOT . SF" as valid short names; using this option does not preserve that
-exception.
-.IP "\fB\-t\fR" 4
+Previous versions of this program exceptionally treated \fIEA DATA . SF\fP and
+\fIWP ROOT . SF\fP as valid short names; using this option does not preserve
+that exception.
+.IP "\fB\-t\fP" 4
```

Comment:
I don't think replacing the quotes with a font instruction is a good idea.  Italic is typically used for placeholders the user needs to replace, which is not the case here.  Literal quotes are fine for strings that are meant verbatim.

Reply:
I changed filenames from quotes to italic in whole manpage for consistency. It is because linux `man-pages(7)` says in section `Formatting conventions (general)` that: Filenames (whether  pathnames, or references to header files) are always in italics. `EA DATA. SF` and `WP ROOT. SF` are really filenames.

And there is a typo, those filenames are `EA DATA. SF` and `WP ROOT. SF` (not `EA DATA . SF`, there is no space before period).

Reply:
Oh, these are filenames?  In that case, you are right, always set filenames with .I (italic).

Regarding spacing: Is there really a space *after* the period?  That would seem very unusual for a filename.

Reply:
Yes, those two files are special FAT12 filenames used by OS/2 for storing extended attributes.  And yes, extension for these file names is ` SF` with leading space. So in full file name is space after period (and not before period). It is unusual but allowed in FAT and used by OS/2.